docs/SPECS.md

```diff
@@ -538,16 +558,16 @@ The relation to determine this mapping is as follow:
 // * round: the round number at which the drand network is at
 // * the time at which this round started
 func CurrentRound(now, genesis int64, period uint32) (round uint64, time int64){
-	if now < genesis {
+    if now < genesis {
         // round 0 is the genesis block: signature is the genesis seed
-		return 0
-	}
-	fromGenesis := now - genesis
-	// we take the time from genesis divided by the periods in seconds, that
-	// gives us the number of periods since genesis.  We add +1 because round 1 
+        return 0
+    }
+    fromGenesis := now - genesis
+    // we take the time from genesis divided by the periods in seconds, that
+    // gives us the number of periods since genesis.  We add +1 because round 1
     // starts at genesis time.
-	round = uint64(math.Floor(float64(fromGenesis)/period)) + 1
-	time = genesis + int64(nextRound*uint64(period.Seconds()))
+    round = uint64(math.Floor(float64(fromGenesis)/period)) + 1
+    time = genesis + int64(nextRound*uint64(period.Seconds()))
```

Comment:
Again, tabs are used by the golang community. There are more instances when its been changed to spaces but I won't comment again :)

Reply:
No problem. Didn't realize it was a Golang specific thing. I'll roll them all back.